build.ps1

```diff
@@ -60,6 +63,10 @@ else {
     } else {
         ExecSafe { & powershell $DotNetInstallFile -InstallDir $DotNetDirectory -Version $DotNetVersion -NoPath }
     }
+    # Octopus Modification
+    # Install the .NET 6.0 runtime as well
+    ExecSafe { & powershell $DotNetInstallFile -InstallDir $DotNetDirectory -Runtime dotnet -Channel 6.0 -NoPath }

```

Comment:
We need this to run the net6.0 tests

Reply:
Would it make sense to update the script comment to capture that?